lib/Conversion/TritonGPUToLLVM/ConvertLayoutOpToLLVM.cpp

```diff
@@ -376,7 +376,7 @@ struct ConvertLayoutOpUsingLinearLayoutsConversion
     // completed before we can remove the layoutIsOK check:
     // 1. Support for AMD's WMMA
     std::function<bool(Attribute)> layoutIsOK = [&](Attribute layout) {
-      if (isa<NvidiaMmaEncodingAttr, AMDMfmaEncodingAttr>(layout)) {
+      if (isa<MmaEncodingTrait>(layout)) {
```

Comment:
There is another Mfma case below (parent of DotOperand). Does it also need considered?

Reply:
It have not been implemented yet. I guess @binarman is assigned.

Reply:
Wmma dot operand is in progress, will make a PR soon.
Some discrepancies in legacy implementation delayed LL PR a little, util I realized what is the ground truth.